components/registry/org.wso2.carbon.registry.extensions/src/main/java/org/wso2/carbon/registry/extensions/utils/CommonConstants.java

```diff
@@ -106,6 +106,8 @@
 
     public static final int NO_OF_DOCUMENTS_ALLOWED = 3;
 
+    public static final String SOURCEURL_PARAMETER_NAME="sourceURL";
```

Comment:
// This constant will use to identify the property key of resources uploaded using URL
will add above comment